ui/components/app/asset-list-item/asset-list-item.js

```diff
@@ -138,7 +138,7 @@ const AssetListItem = ({
       rightContent={
         !isERC721 && (
           <>
-            <i className="fas fa-chevron-right asset-list-item__chevron-right" />
+            <Icon name={ICON_NAMES.ARROW_RIGHT} />
```

Comment:
Same here. It also might be worth checking what the additional className does

Reply:
How do i check what those additional className does.....?

Reply:
Also I don't know how I can run this extension in my local machine to check those things.......
Could you please help.

Reply:
Hey @imdebamrita, having a local build is probably a prerequisite for taking on this task please read through our contributor docs https://github.com/MetaMask/metamask-extension#building-locally